Service/src/routes/stock.py

```diff
@@ -13,127 +13,136 @@
 MONTH = '1m'
 YEAR = '1y'
 
-IEX_DATETIME_FORMAT = '%Y%m%d %H:%M'
+IEX_DATETIME_FORMAT = '%Y-%m-%d %H:%M'
 IEX_DATE_FORMAT = '%Y-%m-%d'
 DATETIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 
 stock_api = Blueprint('stock_api', __name__)
 
-IEX_URL_PREFIX = 'https://api.iextrading.com/1.0/stock/'
+IEX_URL_PREFIX = 'https://cloud.iexapis.com/v1/stock/'
+
+CONFIG_FILE_PATH = './config.json'
+
+# This needs to be optimized so it doesn't open and close the config file every time.
+def getIexToken():
+	try:
+		configFile = open(CONFIG_FILE_PATH, 'r')
```

Comment:
too much of an indent haha

Reply:
yeah... I'll fix this. We do need to update all the files to follow pep8 eventually though.